server/app.js

```diff
@@ -46,6 +46,12 @@ app.use((req, res, next) => {
   next();
 });
 
+app.all(
```

Comment:
Wouldn't putting this above the login and register methods cause those endpoints to not work anymore?

Reply:
@ethanlee16 good catch! 😅the previous change did break it, since it was parsing the request as JSON before it hit the proxy and it was checking for a valid token for the register/login methods.

Solution that is reflected in the next commit:

Since we only need to parse the incoming request as JSON for Login/Register I applied the middleware bodyParser.json() to only register/login